src/test/java/com/hotline/pages/BaseHotlinePage.java

```diff
@@ -0,0 +1,11 @@
+package com.hotline.pages;
+
+import com.hotline.modules.HeaderModule;
+
+public class BaseHotlinePage<T extends BaseHotlinePage> {
```

Comment:
Why do you use generics here?

Reply:
@andyyarych You are right! there is no need in them in BaseHotlinePage and HomePage classes.